src/bot.ts

```diff
@@ -44,6 +46,9 @@ export const startBot = async (): Promise<void> => {
     });
     client.on('error', client.logger.error);
     client.on('ready', initReady);
+    client.on('messageCreate', (message: Message) => {
+      if (client.user) initMessageCreate(client, container.logger, message);
+    });
     client.login();
```

Comment:
Move if(client.user) check out

Reply:
Can't you just delete it

Reply:
Good question for the devs I don't see why we have it but I copied it anyways